ownCloudTests copy-Info.plist

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please use a better file name, without `copy` inside: `ownCloudTests copy-Info.plist` or is it no longer needed, because there is already a file `ownCloudScreenshotsTests/Info.plist`

Reply:
Uploaded accidentally, I have to remove it